test/core/util/fuzzer_corpus_test.cc

```diff
@@ -48,15 +50,24 @@ DEFINE_string(directory, "", "Use this directory as test data");
 class FuzzerCorpusTest : public ::testing::TestWithParam<std::string> {};
 
 TEST_P(FuzzerCorpusTest, RunOneExample) {
+  // Need to call grpc_init() here to use a slice, but need to shut it
+  // down before calling LLVMFuzzerTestOneInput(), because most
```

Comment:
I do not follow the logic here. Is there going to be a problem that we wrap the `LLVMFuzzerTestOneInput` with `grpc_init` and `grpc_shutdown`?

Reply:
The idea is that if a particular implementation needs to test initializing and shutting down gRPC multiple times, or if it needs to do something before the init or after the shutdown, it would not be able to do it if it's called from a context where gRPC is already initialized.

I'm not sure if we actually have any such implementations, but it seems better to be flexible here.